docs/source/onnx.rst

```diff
@@ -161,8 +162,126 @@ list.  The operator set above is sufficient to export the following models:
 * VGG
 * `word_language_model <https://github.com/pytorch/examples/tree/master/word_language_model>`_
 
-The interface for specifying operator definitions is highly experimental
-and undocumented; adventurous users should note that the APIs will probably
+If the wanted operator is standardized in ONNX, it should be easy to add
```

Comment:
This section needs to more clearly state that adding operators is advanced usage; for example, some of the instructions below suggest that you look into generated source files, but those instructions only make sense for users which have done for-source builds.